src/main/java/ch/cern/eam/wshub/core/services/material/PhysicalInventoryService.java

```diff
@@ -0,0 +1,34 @@
+package ch.cern.eam.wshub.core.services.material;
+
+import ch.cern.eam.wshub.core.annotations.Operation;
+import ch.cern.eam.wshub.core.client.InforContext;
+import ch.cern.eam.wshub.core.services.INFOR_OPERATION;
+import ch.cern.eam.wshub.core.services.material.entities.PhysicalInventory;
+import ch.cern.eam.wshub.core.services.material.entities.PhysicalInventoryRow;
+import ch.cern.eam.wshub.core.tools.InforException;
+
+public interface PhysicalInventoryService {
+    @Operation(logOperation = INFOR_OPERATION.INVENTORY_C)
+    PhysicalInventory createPhysicalInventory(InforContext context, PhysicalInventory physicalInventory)
+        throws InforException;
+
+    @Operation(logOperation = INFOR_OPERATION.INVENTORY_R)
+    PhysicalInventory readPhysicalInventory(InforContext context, String code)
+        throws InforException;
+
+    @Operation(logOperation = INFOR_OPERATION.INVENTORY_U)
+    PhysicalInventory updatePhysicalInventory(InforContext context, PhysicalInventory physicalInventory)
+        throws InforException;
+
+    @Operation(logOperation = INFOR_OPERATION.INVENTORY_R)
```

Comment:
Each different operation should have a different key for the logger

Reply:
https://github.com/cern-eam/eam-wshub-core/pull/22/files#diff-6146b6776d5383f5c8a97c71c1f08169R11